lib/msf/core/opt.rb

```diff
@@ -39,13 +39,13 @@ def self.Proxies(default=nil, required=false, desc="A proxy chain of format type
       Msf::OptString.new(__method__.to_s, [ required, desc, default ])
     end
 
-    # @return [OptAddressRange]
-    def self.RHOSTS(default=nil, required=true, desc="The target host(s), range CIDR identifier, or hosts file with syntax 'file:<path>'")
-      Msf::OptAddressRange.new('RHOSTS', [ required, desc, default ], aliases: [ 'RHOST' ])
+    # @return [OptRhosts]
+    def self.RHOSTS(default= nil, required=true, desc="The target host(s) of format 10.10.10.10, cidr identifier 10.10.10.10/24, 'http:<url>' which overrides the RPORT/SSL/TARGETURI values, combine cidr and http with 'cidr:/24:http://<url>', or host file 'file:<path>'")
+      Msf::OptRhosts.new('RHOSTS', [ required, desc, default ], aliases: [ 'RHOST' ])
```

Comment:
TODO: Let's rip out this level of detail, and give a link to the wiki as it clutters up the `help` UI too much now:
![image](https://user-images.githubusercontent.com/60357436/128543649-b3b10e4e-9b8d-4488-9719-a91f21b1cc5c.png)

In the future we could probably add short/long form support for module options, but let's skip that for now :+1:

Reply:
Makes sense just moving to a short description and adding something like "for more valid formats see...", a link would get long, so might be reasonable to just say `RHOSTS documentation`.